cernopendata/modules/fixtures/data/articles/welcome/welcome.md

```diff
@@ -1,14 +1,12 @@
-FIXME PRELIMINARY TEXT
-
-With the growing demand of open science data and increased interest in the CERN Open Data portal, we were working hard on modernizing our web presence. Here, we are telling you more about the changes we made and how you can use the updated portal.
+With the growing demand for open research data and an increased interest in the CERN Open Data portal, we have been working hard on modernising our web presence and services. Here, you can learn more about the changes we have made and how you can use our updated portal.
 
 <p align="center">
 <img src="/static/articles/welcome/header-image.png" width="70%"></p>
 
-CERN Open Data provides content for both education and research. We aim to support high school students and teachers as well as university students and professors. For that purpose, the new design of CERN Open Data is article-driven. Exciting [guides](/search?type=Documentation&subtype=Guide) introduce to physics analyses and make use of [datasets](/search?type=Dataset). Event displays allow to visualise and imagine what happens during collisions of the smallest elements.
+CERN Open Data provides content for both education and research. We aim to support high-school students and teachers as well as university students and professors. For this purpose, the new design of CERN Open Data is article-driven. Detailed [guides](/search?type=Documentation&subtype=Guide) introduce you to physics analyses and make use of [datasets available from this portal](/search?type=Dataset). Event displays allow to visualise collision data and examine what happens when particles interact in high-energy collisions.
+
+Even though enthusiastic members of the wider public as well educators in particular have been on our mind when we redesigned the portal, we did not forget about researchers. It’s quite the opposite actually: our new search features and updated facets allow for an even more efficient and effective access to relevant [datasets](/search?type=Dataset). We are proud that data released on CERN Open Data led to the [first publication](https://arxiv.org/abs/1704.05842) of a research paper using LHC data by researchers not belonging to any LHC collaboration. This is truly open science!
```

Comment:
Can we re-shuffle the phrase a little bit? To me the sentence is confusing. Something like: We are proud that previous data releases on CERN Open Data  have already yielded new results: first publications of research using LHC data written by researchers not belonging to any LHC collaboration came out in 2017. [probably too long, sorry]

Reply:
Yes, I found the sentence clunky as well when I wrote it, but I pushed this commit in a hurry. Will update. Thanks!

Reply:
How about (please read the whole paragraph):

> Even though enthusiastic members of the wider public as well educators in particular have been on our mind when we redesigned the portal, we have made sure that our portal caters to researchers as well. Our new search features and updated facets allow for an even more efficient and effective access to relevant [datasets](/search?type=Dataset). We are proud that our previous data releases have already yielded new results: 2017 saw the publication of the [first scientific paper](https://arxiv.org/abs/1704.05842) using LHC data that was authored by external researchers, that is ones not belonging to an LHC collaboration.

Or…

> […] We are proud that our previous data releases have already yielded new results: 2017 saw the publication of the [first scientific paper](https://arxiv.org/abs/1704.05842) using LHC data that was authored by researchers not belonging to an LHC collaboration.

Reply:
I like the first one. But re-reading this, I realize that I would like citizen science/educators/researchers on the same level. 
Perhaps like "While we had citizen scientists, the wider public, as well educators in mind when we redesigned the portal, we have of course also made sure that our portal caters to researchers interested in these unique materials.
Feel free to change of course!

Reply:
I don't necessarily want to draw a distinction between "citizen scientists" and "academic researchers". But I like your overall restructuring.

> While we had educators, students and the wider public in mind when we redesigned the portal, we have of course also made sure that we cater to the broader research community that may be interested in these unique materials.

?